remix/app/lib/handler/app.handler.test.ts

```diff
@@ -17,10 +17,14 @@ jest.mock('../../config/auth.server', () => ({
   },
 }));
 
-const mockRequest = {};
-const mockCallback = jest.fn();
-
 describe('appHandler', () => {
+  const mockRequest: any = {
+    headers: {
+      get: jest.fn(() => 'Cookie Value'),
```

Comment:
In the last implementation, I changed the toast to use `localStorage` instead of the `Cookie`, and the user is still on the `Cookie`.

For that one, I forgot the remove it.

I will remove it.

Reply:
Removed here: [f078d2a](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/f078d2aa6e5b90e3f455452ffa7e67ef3fd64881)